crates/net/test-utils/src/testnet.rs

```diff
@@ -52,22 +52,28 @@ where
         Ok(this)
     }
 
+    /// Return a mutable slice of all peers.
     pub fn peers_mut(&mut self) -> &mut [Peer<C>] {
         &mut self.peers
     }
 
+    /// Return a slice of all peers.
```

Comment:
I would prefer all these to remain under net/ and have them exported via a test-utils feature?

Let's do these as smaller refactors over the entire PR

Reply:
that makes sense, we can create a `network::test_utils` module under a `test-utils` feature